### PR TITLE
[LOOP-1558] Updated pod reservoir display

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-1558

updated display of pod reservoir remaining.

Also includes:
- update to allow the entry of the magic number in the mock pod setting reservoir remaining field
- updated reservoir mask image
- updated reservoir image

### CODE REVIEW

File: DashKit/ReservoirLevel.swift [hunk 1]
```diff
@@ -14,14 +14,16 @@ public enum ReservoirLevel: RawRepresentable, Equatable {
...
+    case aboveThreshold // the threshold is 50 units remaining
 
     public func asPercentage() -> Double? {
         switch self {
         case .aboveThreshold:
+            // reservoir has more than 50 units remaining
             return nil
         case .valid(let value):
-            return min(1, max(0, value / Pod.reservoirCapacity))
+            // design requested that reservoir level start display at 50%. This means that 50 units remaining = 50%, 25 units = 25% and so on.
```
Comment (ps2) [inline DashKit/ReservoirLevel.swift:25]: The Omnipod reservoir is 200U.  With this change, the reservoir icon will show 50% when the reservoir is actually 25% full. Is that really the intention here?

**Review by ps2 — COMMENTED** [review]
I'm not sure about the visual percentage change here.

File: DashKit/ReservoirLevel.swift (hunk 1 above)
Comment (nhamming) [reply DashKit/ReservoirLevel.swift:25]: correct. This was requested from design. I'll put it on my standup topic list to confirm.

Comment (ps2) [reply DashKit/ReservoirLevel.swift:25]: Would be good to hear the reasoning here. I don't think the iconography needs to match the physical device with complete accuracy, but this showing double the reality.

**Comment by nhamming** [comment]
Some of this is coming with [LOOP-1409](https://github.com/tidepool-org/DashKit/pull/72). The questionable part that needs design input will wait until the new year.

**Pull request closed without merging**